app/views/students/status_updates/index.html.slim

```diff
@@ -15,13 +16,12 @@ table.table.table-striped.table-bordered.table-condensed
       th
 
   tbody
-    - @status_updates.each do |status_update|
+    - status_updates.each do |status_update|
       tr
-        td = l status_update.created_at
+        /date format is Weekday, MM dd, hh:mm
+        td = l status_update.created_at, :format => ('%A, %B %e, %H:%M (utc)')
```

Comment:
Can you use or define a date/datetime format in `config/locales/en.yml`? That way it can be re-used and we don't scatter the app with too many different ways of printing date formats.